src/overlays/actors/ovl_En_Invadepoh_Demo/z_en_invadepoh_demo.c

```diff
@@ -15,14 +27,72 @@ void EnInvadepohDemo_Destroy(Actor* thisx, PlayState* play);
 void EnInvadepohDemo_Update(Actor* thisx, PlayState* play);
 void EnInvadepohDemo_Draw(Actor* thisx, PlayState* play);
 
-void func_80C19AB4(EnInvadepohDemo* this, PlayState* play);
-void func_80C19D00(EnInvadepohDemo* this, PlayState* play);
-void func_80C19D48(EnInvadepohDemo* this, PlayState* play);
-void func_80C19E04(EnInvadepohDemo* this, PlayState* play);
-void func_80C19EC0(EnInvadepohDemo* this, PlayState* play);
-void func_80C19F7C(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_DoNothing(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Alien_Init(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Romani_Init(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_Init(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Ufo_Init(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_CowTail_Init(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_Destroy(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_CowTail_Destroy(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Alien_Idle(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Alien_FollowPath(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Romani_Idle(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Romani_FollowPath(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_Idle(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_FollowPath(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Ufo_Idle(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Ufo_FollowPath(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_SelectCueAction(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_CowTail_Idle(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Alien_WaitForObject(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Romani_WaitForObject(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_WaitForObject(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_CowTail_WaitForObject(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Alien_Draw(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Romani_Draw(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Cow_Draw(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_Ufo_Draw(EnInvadepohDemo* this, PlayState* play);
+void EnInvadepohDemo_CowTail_Draw(EnInvadepohDemo* this, PlayState* play);
+
+#define DRAW_FLAG_SHOULD_DRAW 1
+#define EN_INVADEPOH_DEMO_CUE_ID_NONE -1
+
+typedef enum {
+    /* 0 */ EN_INVADEPOH_DEMO_ALIEN_CUE_ID_DO_NOTHING,
+    /* 1 */ EN_INVADEPOH_DEMO_ALIEN_CUE_ID_IDLE,
```

Comment:
```suggestion
    /* 0 */ EN_INVADEPOH_DEMO_ALIEN_CUEID_DO_NOTHING,
    /* 1 */ EN_INVADEPOH_DEMO_ALIEN_CUEID_IDLE,
```
Just a small comment, I've been doing `CUEID` as 1 word because it seems easier to read in a lot of cases, despite `CueId` being two words, although I'm not convinced that's better, and it may be better as the way you have it. Thoughts?

Reply:
Yeah, that makes sense. `ID` by itself doesn't really mean anything; you need to have the `CUE` there for it to make sense. So it's logical to combine them into one "unit", if that makes sense.